### PR TITLE
Zandalari Set

### PR DESCRIPTION
**Pull request by panpoleon**
## Changelog:
- Added the Rastakhan crown and armor from WoW as emperor clothes for the trolls.

# Tasks:
- [x] Fix errors mentioning `zanda`

# How to test:
Check how it looks on Rastakhan and his ancenstors.

### CODE REVIEW

File: common/genes/05_genes_special_accessories_clothes.txt [hunk 1]
```diff
@@ -1361,6 +1361,18 @@ special_genes = {
 				boy = male
```
Comment (zumbak04) [inline common/genes/05_genes_special_accessories_clothes.txt:1361]: See these black lines? Looks like displaced normal map to me.
<details>
<summary>Click to expand</summary>

image

</details>

**Review by zumbak04 — CHANGES_REQUESTED** [review]

**Review by TheDrPibb1999 — CHANGES_REQUESTED** [review]
In the war screen the set almost looks tan 
Capture

File: common/genes/05_genes_special_accessories_clothes.txt (hunk 1 above)
Comment (zumbak04) [inline common/genes/05_genes_special_accessories_clothes.txt:1361]: Could you resave the crown textures as well? It has no alpha too,
<details>
<summary>Click to expand</summary>

image

</details>

**Review by zumbak04 — CHANGES_REQUESTED** [review]

**Review by TheDrPibb1999 — COMMENTED** [review]
In the war screen the feathers clip through the chest 
Capture

File: common/genes/05_genes_special_accessories_clothes.txt (hunk 1 above)
Comment (panpoleon) [reply common/genes/05_genes_special_accessories_clothes.txt:1361]: done

[the remark this answers is not in the document]
Comment (panpoleon) [reply common/genes/05_genes_special_accessories_clothes.txt:1361]: Im not sure if I can do anything about this one :/

File: common/genes/05_genes_special_accessories_clothes.txt [hunk 2]
```diff
@@ -1362,7 +1362,7 @@ special_genes = {
 				girl = female
```
Comment (zumbak04) [inline common/genes/05_genes_special_accessories_clothes.txt:1362]: Still persist.
<details>
<summary>Click to expand</summary>

image

</details>

**Review by zumbak04 — CHANGES_REQUESTED** [review]

File: common/genes/05_genes_special_accessories_clothes.txt (hunk 2 above)
Comment (zumbak04) [reply common/genes/05_genes_special_accessories_clothes.txt]: Nope, it still has problems. Check how it looks in the Portrait Editor before pushing it.
<details>
<summary>Click to expand</summary>

image

</details>

File: common/genes/05_genes_special_accessories_clothes.txt [hunk 3]
```diff
@@ -1421,6 +1421,17 @@ special_genes = {
 				boy = male
```
Comment (zumbak04) [inline common/genes/05_genes_special_accessories_clothes.txt:1421]: Noticed shoulders and clothes clip.
<details>
<summary>Click to expand</summary>

image

</details>

**Review by zumbak04 — CHANGES_REQUESTED** [review]

**Review by zumbak04 — APPROVED** [review]
LGTM! Good job!

**Review by Remerod — CHANGES_REQUESTED** [review]
It looks pretty good, On all of them. With one exception though. The throat is clipping through the.. throat/neck part of the Armor. I believe if this is fixed the armor would be good to go.
<details>
<summary>Click to expand</summary>

image

image

image

</details>

**Comment by zumbak04** [comment]
> It looks pretty good, On all of them. With one exception though. The throat is clipping through the.. throat/neck part of the Armor. I believe if this is fixed the armor would be good to go.

Should be fixed in e20001117e6939c9c4ca786a5d472862342c21c1

**Review by Remerod — CHANGES_REQUESTED** [review]
While it isn't as notable as before it is still there, while you don't see it at first you can still see something that looks like a double chin and when you've seen it it's there. On a Positive Note, I can only see it on Rastakhan. It is probably because he is bigger then the rest.
<details>
<summary>Click to expand</summary>

image

image

</details>

**Comment by zumbak04** [comment]
> While it isn't as notable as before it is still there, while you don't see it at first you can still see something that looks like a double chin and when you've seen it it's there. On a Positive Note, I can only see it on Rastakhan. It is probably because he is bigger then the rest.

Should be fixed in b6ec5f79b8912fc7e90b625cf0ffb14050b9f664.

**Review by Remerod — COMMENTED** [review]
At first i thought this was clipping again , but it is probably just his throat that is hanging on it. I think it looks ok (not perfect though) So if you folk thinks it looks also ok I'll approve this. leave a comment or a thumbs up or something and I'll know.
<details>
<summary>Click to expand</summary>

image

</details>
It looks worse on the portrait on the red arrow then it does on the purple arrow (barbershop)

**Comment by zumbak04** [comment]
> I think it looks ok (not perfect though)

What do you mean?

**Comment by Remerod** [comment]
I mean that the throat looks like it's clipping through the armor when i it might just hang over it as in the pic i showed, it does look less weird on the purple arrow then the red one. It looks ok on Rastakhan but it look's better on his ancestors

**Comment by zumbak04** [comment]
> I mean that the throat looks like it's clipping through the armor when

But you see there's no clipping though?

**Comment by Remerod** [comment]
I think there is a tiny tiny part that might be clipping, but i think it's ok as it is now

**Comment by zumbak04** [comment]
> I think there is a tiny tiny part that might be clipping, but i think it's ok as it is now.

Can't confirm. I checked in the Portrait Editor, there's no clipping.

<details>
<summary>Click to expand</summary>

image

</details>

**Review by Remerod — APPROVED** [review]
It is this thick throat-chin thing that made it look like clipping. But i guess it is not, and it looks fine. I think it all looks good! Good Work!
<details>
<summary>Click to expand</summary>

image

</details>

**Pull request merged**